cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,50 @@
+import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
+import faker from 'faker';
+
 /// <reference types="cypress" />
 
-describe('', () => {
-  before(() => {
+describe('Order - User is able to order the product', () => {
+  const homePage = new HomeAndCataloguePageObject();
 
+  beforeEach(() => {
+    cy.viewport(1000, 600);
+    homePage.visit();
   });
 
-  it('', () => {
+  it('should allow the user to order the product', () => {
```

Comment:
you don't use POM in your test. Check the [theory](https://mate.academy/en/learn/javascript-testing/cypress?section=theory&videoId=2134&testTaskSlug=qa_cypress_demoblaze_pom_basics)